jscripts/thread.js

```diff
@@ -6,6 +6,30 @@ var Thread = {
 			Thread.initQuickReply();
 			Thread.initMultiQuote();
 
+			if(thread_deleted == "1")
+			{
+				$("#quick_reply_form, .new_reply_button, .thread_tools, .inline_rating").hide();
+				$("#moderator_options_selector option.option_mirage").attr("disabled","disabled");
+			}
+
+			visible_replies = parseInt(visible_replies);
```

Comment:
When using `parseInt`, it's best to always specify a base as otherwise JS tries to do some "magic".

```suggestion
			visible_replies = parseInt(visible_replies, 10);
```

Reply:
Done.